formats/dot/internal/errors/errors.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// generated by gocc; DO NOT EDIT.
+// Code generated by gocc; DO NOT EDIT.
```

Comment:
Hmm.

It looks like it hides some of the files, even when a copyright header was preceding the "Code generated" comment. For instance, in my browser ` formats/dot/internal/lexer/transitiontable.go` is hidden, even though it contains:

```go
// Copyright ©2017 The gonum Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Code generated by gocc; DO NOT EDIT.
```

https://github.com/gonum/graph/pull/198/files

Looking a bit closer, it seems that was because of the length of the file, as ` formats/dot/ast/ast.go` was also hidden by default, but this file is hand written and does not contain a "Code generated" comment.